src/code/z_camera_data.inc.c

```diff
@@ -2464,59 +2464,115 @@ s32 sDemo5PrevSfxFrame = -200;
 
 // target is player, far from eye
 OnePointCsFull D_8011D6AC[] = {
-    { // initflags & 0x00FF (at): 2, atTarget is view lookAt + atInit
-      // initFlags & 0xFF00 (eye): none
-      // action: 15, copy at, eye, roll, fov to camera
-      // result: eye remains in the same locaiton, at is View's lookAt
-      0x8F,
-      0xFF,
-      0x0002,
-      0x0001,
-      0x0000,
-      60.0f,
-      1.0f,
-      { 0.0f, 0.0f, 0.0f },
-      { 0.0f, 0.0f, 0.0f } },
-    { // initFlags & 0x00FF (at): 3, atTarget is camera's current at + atInit
-      // initFlags & 0xFF00 (eye): 3, eyeTarget is the camera's current eye + eyeInit
-      // action: interplate eye and at.
-      // result: eye and at's y interpolate to become +20 from their current location.
-      0x81,
-      0xFF,
-      0x0303,
-      0x0013,
-      0x0000,
-      45.0f,
-      1.0f,
-      { 0.0f, 20.0f, 0.0f },
-      { 0.0f, 20.0f, 0.0f } },
-    { // initFlags & 0x00FF (at): 0 none
-      // initFlags & 0xFF00 (eye): 0 none
-      // action: 18, copy this camera to default camera.
-      0x12,
-      0xFF,
-      0x0000,
-      0x0001,
-      0x0000,
-      60.0f,
-      1.0f,
-      { -1.0f, -1.0f, -1.0f },
-      { -1.0f, -1.0f, -1.0f } },
+    {
+        // initflags & 0x00FF (at): 2, atTarget is view lookAt + atInit
+        // initFlags & 0xFF00 (eye): none
+        // action: 15, copy at, eye, roll, fov to camera
+        // result: eye remains in the same locaiton, at is View's lookAt
+        ONEPOINT_CS_ACTION(ONEPOINT_CS_ACTION_ID_15, false, true),
+        0xFF,
+        0x0002,
+        0x0001,
```

Comment:
Not really related to this PR but these `timerInit` values should probably be decimal like the cutscenes defined in `z_onepointdemo_data.inc.c`

Reply:
Makes sense. I made this data in camera_data consistent with onepointdemo_data